<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit_c_a</class>
 <widget class="QWidget" name="credit_c_a">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(245, 255, 137);
	color: black;
}</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
}</string>
     </property>
     <property name="text">
      <string>Pay by mounth</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
}</string>
     </property>
     <property name="text">
      <string>Overpay</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_ovp">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: white;
	color: black;
	border: 1px solid grey;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: black;
}</string>
     </property>
     <property name="text">
      <string>Full payment</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_fp">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	background-color: white;
	color: black;
	border: 1px solid grey;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="styleSheet">
      <string notr="true">QTextBrowser {
	background-color: white;
	color: black;
	border: 1px solid grey;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
